tasties/urls.py

```diff
@@ -25,5 +27,8 @@
     path('login/', views.login_user, name="login"),
     path('register/', views.register, name="register"),
     path('logout/', views.logout_user, name="logout"),
-    path('view_recipe/<int:recipe_id>/', views.view_recipe, name="view_recipe")
+    path('view_recipe/<int:recipe_id>/', views.view_recipe, name="view_recipe"),
+    path('create_recipe/', views.create_recipe, name="create_recipe")
 ]
+
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Could you please explain why you need this?

Reply:
@GilShechter Would be in a better position to explain this, since this is part of his contribution to this pr.

Reply:
It's necessary for file uploading, which takes place in the `CreateRecipeForm`.
We want the user to upload an image for his recipe, so this concatenation to the `urlpatterns` navigates the file to the desired path.
You can read about it [here](https://docs.djangoproject.com/en/2.0/howto/static-files/#serving-files-uploaded-by-a-user-during-development).

Reply:
Thanks. Keep in mind this comment in the doc `This is not suitable for production use! `
It ok for our course use case though